plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmCaseProgressWidgetAction.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2018 Entando Inc..
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.entando.entando.plugins.jpkiebpm.apsadmin.portal.specialwidget;
+
+import com.agiletec.aps.system.exception.ApsSystemException;
+import com.agiletec.apsadmin.portal.specialwidget.SimpleWidgetConfigAction;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.commons.lang3.StringUtils;
+import org.entando.entando.plugins.jpkiebpm.aps.system.services.bpmwidgetinfo.IBpmWidgetInfoManager;
+import org.entando.entando.plugins.jpkiebpm.aps.system.services.kie.CaseManager;
+import org.entando.entando.plugins.jpkiebpm.aps.system.services.kie.IKieFormManager;
+import org.entando.entando.plugins.jpkiebpm.aps.system.services.kie.IKieFormOverrideManager;
+import org.entando.entando.plugins.jpkiebpm.aps.system.services.kie.model.kieProcess;
+
+/**
+ *
+ * @author own_strong
+ */
+public class BpmCaseProgressWidgetAction extends SimpleWidgetConfigAction {
+
+//    public static final String SUCCESS = "success";
+    private CaseManager caseManager;
+    private IKieFormManager formManager;
+    private IKieFormOverrideManager kieFormOverrideManager;
+    private IBpmWidgetInfoManager bpmWidgetInfoManager;
+    String processPath;
+    String processPathDefaultValue;
+    String casesDefinitions;
+    List<String> process;
+
+    String _frontEndMilestonesData;
```

Comment:
Remove underscores in variable names. Its all over the codebase but trying to normalize to java standards for new stuff or changed code.